docs/features/pdb-compilation-options.md

```diff
@@ -270,7 +270,7 @@ Example:
 | output-kind            | string                                  | require   | The value passed to `/target` |
 | platform               | string                                  | require   | The value passed to `/platform` |
 
-#### Options For CSharp
+#### Options For C\#
```

Comment:
Why did you escape the `#` here but not in the other changes?

Reply:
@jaredpar See https://github.com/dotnet/roslyn/pull/57427#discussion_r738029048